WebRoot/skins/_base/base4/skin.css

```diff
@@ -1066,3 +1066,41 @@ div.ZmHtmlEditor .TxtAreaDrag {
 .ZmConvDoublePaneView .Row-selected .ZmMsgListDate {
     background-color: @ListItemBgColor-selected@ !important;
 }
+
+/*Circular Progress bar styling*/
+svg.progress-circle {
+	@FullWidth@
+}
+
+svg.progress-circle text {
+	fill: @TxtC@;
+}
+
+svg.progress-circle circle.bg {
+	stroke: #e5e5e5;
```

Comment:
good to have some properties in skin.properties to change styling of progress bars